TODO/what-i-hate-in-kotlin.md

```diff
@@ -133,21 +122,11 @@ val list = if(student != null && student.passing) {
 }
 ```
 
-I do not judge, but the fact is that implementation that is strongly using all Kotlin extensions are hard and unintuitive for people
-who are not Kotlin developers. This kind of features are making Kotlin harder and harder for beginners. The big change comes with
-Kotlin Coroutines. It is great feature. When I started learning it, then through whole day I was repeating “incretable” and “wow”.
-It is awesome how Kotlin Coroutines are making multithreading so simple. I feel that this should be designed this way from the beginning of programming.
-Still, it is complex to understand Kotlin Coroutines, and it is far away from how it is implemented in other technologies.
-If community will start strongly using Coroutines, then it will be another barrier for programmers from other languages to jump in. This
```

Comment:
> If community will start strongly using Coroutines, then it will be another barrier for programmers from other languages to jump in.This leads to isolationism. And I feel that it is too early for that.

如果社区开始广泛地使用协程，它又会成为其他语言的开发者入门的另一个障碍。